tgui/packages/tgui/interfaces/Fabrication/MaterialAccessBar.tsx

```diff
@@ -43,15 +44,15 @@ const LABEL_FORMAT = (value: number) => formatSiUnit(value, 0);
  * the ore silo. Has pop-out docks for each material type for ejecting up to
  * fifty sheets.
  */
-export const MineralAccessBar = (props: MineralAccessBarProps, context) => {
+export const MaterialAccessBar = (props: MaterialAccessBarProps, context) => {
```

Comment:
Was the file renamed to match? Can't tell from the phone UI.

Reply:
Yep!